invenio_drafts_resources/records/systemfields/versions.py

```diff
@@ -199,8 +199,8 @@ def post_create(self, record):
         # The parent record is created on pre_create.
         versions = self.obj(record)
         if self._create and self._set_next:
-            # if fork is none - it's a new?
-            versions.set_next()
+            if not record.is_published:
```

Comment:
This is the fix. When interacting with the site in the browser it fixes the issue.